server/src/main/java/org/opensearch/tasks/TaskStatsType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tasks;
+
+public enum TaskStatsType {
+    MEMORY("memory"),
+    CPU("cpu");
```

Comment:
What do these stats mean? Like is the memory value a percentage or bytes? Maybe these answers are obvious to everyone but me :)